docs/book/src/introduction.md

```diff
@@ -0,0 +1 @@
+{{#include ../../../README.md:2:}}
```

Comment:
Just saw that we're already mentioning CAPI v0.4 in the top-level README.md (Compatiblity with Cluster API...)

I think we should drop this row/column because v0.4 is not released yet and it's really confusing.

(last estimation was that v0.4 will be released in June)

Reply:
I think we should drop the `Documentation` section in the README.md

That's really confusing in the book:
`Documentation can be found in the /docs directory`